api/compression.md

```diff
@@ -11,14 +11,36 @@ policy](/compression/add_compression_policy/) for when to compress chunks.
 Advanced usage of compression allows users to [compress chunks
 manually](/compression/compress_chunk), instead of automatically as they age.
 
+Compressed chunks have the following limitations:
+  - Constraints are restricted to compression `SEGMENTBY` columns when
+    applicable. Otherwise they are unsupported.
+  - `ROW LEVEL SECURITY` is not supported.
+
 ### Restrictions
 
-The current version does not support altering or inserting data into compressed
-chunks. The data can be queried without any modifications, however if you
-need to backfill or update data in a compressed chunk you will need to
+The current version does not fully support altering or inserting data into
+compressed chunks. The data can be queried without any modifications, however
+if you need to backfill or update data in a compressed chunk you will need to
 decompress the chunk(s) first.
 
 Starting with TimescaleDB 2.1, users have the ability to modify the schema
 of hypertables that have compressed chunks.
 Specifically, you can add columns to and rename existing columns of 
-such compressed hypertables.
\ No newline at end of file
+such compressed hypertables.
+
+Starting with TimescaleDB 2.3, users have the ability to insert data into
+compressed chunks, and enable compression policies on distributed hypertables.
+
+Altering data of compressed chunks still has some limitations:
+  - You cannot execute `UPDATE` or `DELETE` statements on compressed chunks.
+  - `INSERT` is not fully supported on compressed chunks:
+    - You cannot use the `ON CONFLICT` clause (upserts).
+    - You cannot use the `OVERRIDING` clause.
+    - You cannot use the `RETURNING` clause.
+  - Triggers are not fully supported when inserting into compressed chunks:
+    - You cannot use `AFTER INSERT` row-level triggers (`FOR EACH ROW`).
+  - Constraints are not fully supported:
```

Comment:
The constraints section is not specific to 2.3 (Constraints are not fully supported and what follows it). It should be moved to the earlier Restrictions section (line 15)